run_exp.py

```diff
@@ -34,18 +34,20 @@
 makefolder(trace_folder)
 makefolder(log_folder)
 
-t = traceloader(trace_list, trace_folder, 1, 1)
-trace_info=t.retinfo()
+traceloader = TraceLoader(trace_list, trace_folder, 1, 1, 1)
+traceloader.load()
+trace_info = traceloader.read_trace_info()
 # trace_info = json.load(
 #     open(os.path.join(trace_folder, 'trace_info.json'), encoding='utf-8'))
 clear_folder(log_folder)
 
 print('Running iperf server...')
-iperf_server = subprocess.Popen(f'iperf -s -p {IPERF_PORT} >./{log_folder}/iperf_server.log',
-                                shell=True,
-                                stdout=subprocess.PIPE,
-                                stderr=subprocess.PIPE,
-                                encoding="utf-8")
+iperf_server = subprocess.Popen(
+    f'iperf -s -p {IPERF_PORT} >./{log_folder}/iperf_server.log',
+    shell=True,
+    stdout=subprocess.PIPE,
+    stderr=subprocess.PIPE,
+    encoding="utf-8")
```

Comment:
Better not change this. Only concern about the code block you must change. If you think this block of code should be formatted, leave it another commit or pr.